doc/api/process.md

```diff
@@ -1489,8 +1489,10 @@ Android)
 
 * {Stream}
 
-The `process.stderr` property returns a [Writable][] stream connected to
-`stderr` (fd `2`).
+The `process.stderr` property returns a stream connected to
+`stderr` (fd `2`). It is a `net.Socket` (which inherits from 
+`stream.Duplex`) unless fd `2` refers to a file, in which case it is 
```

Comment:
It would be worthwhile to have this (and the other reference below) be a link to the `Duplex` doc